src/main.cpp

```diff
@@ -2461,8 +2461,7 @@ enum FlushStateMode {
  * if they're too large, if it's been a while since the last write,
  * or always and in all cases if we're in prune mode and are deleting files.
  */
-bool static FlushStateToDisk(CValidationState &state, FlushStateMode mode) {
-    const CChainParams& chainparams = Params();
+bool static FlushStateToDisk(CValidationState &state, const CChainParams& chainparams, FlushStateMode mode) {
```

Comment:
If you don't want to do that manually (or remember the exact "rules"), you can just do `$ git diff HEAD~ -U0|cfd`. Where `cfd` is an alias for  `./contrib/devtools/clang-format-diff.py -p1 -i -v`.